src/utils/roofline_calc.py

```diff
@@ -195,11 +195,21 @@ def calc_ai(sort_type, ret_df):
     df = df.sort_values(by=["Kernel_Name"])
     df = df.reset_index(drop=True)
 
-    total_flops = valu_flops = mfma_flops_bf16 = mfma_flops_f16 = mfma_iops_i8 = (
+    total_flops = (
```

Comment:
You probably don't need all these whitespace / formatting changes in the same commit?
Unless this is this linter throwing issues?

Reply:
Right. That was from my local yaml formatter. I should not avoid it as it is a noise for code reviewing.